coderd/database/dbauthz/methods.go

```diff
@@ -0,0 +1,1622 @@
+package dbauthz
```

Comment:
@Emyrk thoughts on calling this `querier.go` instead of `methods`? Methods feels very Java-esque to me.

Reply:
Calling them "methods" is [technically correct](https://go.dev/ref/spec#Method_declarations), but I have no preference here.

Reply:
Yea, I was going with the golang naming of methods being functions attached to a struct. In this case, the primary structure of the package.